pylana/filter.py

```diff
@@ -0,0 +1,193 @@
+import time
+from dateutil.parser import parse
+
+def combine_filters(*filters) -> list:
+    """
+    Create a list of combined filters.
```

Comment:
This function only works for lists and variables. Passing other [iterables](https://docs.python.org/3/glossary.html) will break it. With the current code we can check whether the type of the passed arguments is either `collections.abs.Iterable` or not instead of `list`. This way the passed filters can also be in lists and generators. If we want to be really fancy (but also potentially slightly obscure), we can use the one liner

```python
from collections.abc import Iterable
from functools import reduce
from operator import add

def combine(*filters):
    return reduce(
        add,
        [
            list(a) if isinstance(a, Iterable) else [a]
            for a in  filters
        ]
    )
```

It is not very efficient (there is a bit of casting (everything needs to become a list) involved and we run twice through the passed arguments) , but I thought to post it here, since it shows some nice functional features of python.